coding-guides/making-your-own-starboard.md

```diff
@@ -218,15 +218,15 @@ module.exports = class {
     if (message.author.id === user.id) return;
     if (reaction.emoji.name !== '⭐') return;
     const { starboardChannel } = this.client.settings.get(message.guild.id);
-    const starChannel = message.guild.channels.find(channel => channel.name == starboardChannel)
+    const starChannel = message.guild.channels.cache.find(channel => channel.name == starboardChannel)
     if (!starChannel) return message.channel.send(`It appears that you do not have a \`${starboardChannel}\` channel.`); 
     const fetchedMessages = await starChannel.fetchMessages({ limit: 100 });
```

Comment:
> 
> 
> Doesn't v12 use something like channel.messages.fetch now?

Reply:
nope

Reply:
Looking at the official documentation for _version 12_, they do not have a `fetchMessages` method anymore and Zinc is correct, it's been changed to `channel.messages.fetch`